accounts/accountsRouter.js

```diff
@@ -43,6 +44,8 @@ router.get("/:id", async (req, res) => {
 
 router.put("/:id", async (req, res) => {
   const account = req.body;
+  const hash = bcrypt.hashSync(account.password, 10);
+  account.password = hash;
```

Comment:
This could probably be moved to a helper function for other times where the password should be hashed in an object.